src/service/fixed-layer.js

```diff
@@ -879,6 +972,24 @@ class TransferLayerBody {
     return this.layer_;
   }
 
+  /** @override */
+  setLightboxMode(on) {
+    this.vsync_.mutate(() => {
+      const root = this.getRoot();
+      if (this.isLightboxExperimentOn_) {
+        if (on) {
+          root.setAttribute('i-amphtml-lightbox', '');
```

Comment:
It seems we could just always use this attribute to do the hiding/unhidding, and only add `.i-amphtml-lightbox-element` when the experiment is enabled? I would rename the attr to something more generic.